test/tests/time_integrators/multi_stage_time_integrator/unconverged_1st_stage.i

```diff
@@ -0,0 +1,87 @@
+# This test is designed to check that a time step solve should stop if *any*
+# time integrator solve stage fails, not just the *last* stage. If a time
+# integrator does not check convergence per stage, then a time step proceeds
+# past intermediate stages without checking nonlinear convergence. This test
+# is designed to check that the 2nd stage is never even entered by making it
+# impossible for the first stage to converge.
+#
+# Note that for this time integrator, which is fully explicit, non-time kernels
+# should have "implicit" set to "false"; however, this test is only checking
+# a solve is aborted after the first stage failure, so the time integration
+# scheme itself is irrelevant.
```

Comment:
The danger here is that somebody will find this input file and will copy and paste it and will complain that it does do the right thing for `explicit-tvd-rk2`. People, do not read. So, you really want to set things up properly in a test.

Reply:
Ok, I'll use an implicit time integrator instead so that I avoid having to bastardize the time integration.